lighthouse-core/audits/byte-efficiency/uses-request-compression.js

```diff
@@ -40,8 +41,12 @@ class ResponsesAreCompressed extends ByteEfficiencyAudit {
   static audit_(artifacts) {
     const uncompressedResponses = artifacts.ResponseCompression;
 
+    function skipUnrelatedResponses(record) {
+      return URL.getProtocol(record.url) !== CHROME_EXTENSION_PROTOCOL;
```

Comment:
> we recently discovered that calling `new URL` in a loop on the network records can be crazy slow due to data URIs

gah, oh yeah. I wonder if we can automate testing for audits that will run into this.

We could at least generate a network records fixture (or devtoolsLog now, I guess), that would make it easy to test an audit with many data URIs (and add it to a "writing an audit protips" section in CONTRIBUTING). With @paulirish's desire to individually annotate tests with timeouts, it would be a red flag if an individual test was taking more than two seconds to parse the test fixture.